packages/dart_frog_auth/lib/src/dart_frog_auth.dart

```diff
@@ -47,27 +47,42 @@ Future<bool> _defaultApplies(RequestContext context) async => true;
 /// [RequestContext]. If the function returns false, the middleware will not
 /// apply to the route and the call will have authentication validation.
 Middleware basicAuthentication<T extends Object>({
-  required Future<T?> Function(String, String) userFromCredentials,
+  Future<T?> Function(String, String)? userFromCredentials,
```

Comment:
Note: Although I changed this to not be required anymore, this will not be a breaking change since anyone who was using the middleware will already have this informed, so the assert bellow will not throw.